Makefile

```diff
@@ -158,7 +159,7 @@ endef
 
 .PHONY: test-ci
 test-ci:
-	go test -v -coverprofile=coverage.txt ./...
+	go test -v -coverprofile=coverage.txt ./... && sed -i '/^github.com\/jaegertracing\/jaeger-idl\/model\/v1\/model.pb.go/d' coverage.txt
```

Comment:
I think in the main repo we achieve this with an exclusion rule in `.codecov.yaml`